src/pfe/file-watcher/server/test/functional-test/configs/project.config.ts

```diff
@@ -36,11 +41,27 @@ export const projectCapabilities: any = {
             "controlCommands": ["restart"]
         },
         "kube": project.defaultProjectCapabilities
+    },
+    "spring": {
+        "local": {
+            "startModes": ["run", "debug", "debugNoInit"],
+            "controlCommands": ["restart"]
+        },
+        "kube": project.defaultProjectCapabilities
     }
 };
 
+// auto build enable/disable event capabilities: some projects emit the project changed event and some don't do anything
+export const autoBuildEventCapabailities: any = {
```

Comment:
Should we be fixing this in our source code rather than working around it in the tests? I mean, if some projects dont emit events for the same action, then its a bug, is it not? 😆

Reply:
Not really. This happens on disable and enable auto builds on node projects because of nodemon like Stephanie brought it up in today's meeting. It is not a bug on the source side. On the other having it as capabaility config is nice because in the future if any other project type has a functionaility like how nodemon does for node, it can be applied to it.

Reply:
Thx for the explanation.